### PR TITLE
[13.0] mig project_task_material_stock

### PR DESCRIPTION
**Pull request by jobiols**
Hello @rvalyi Thank you for your support. I decided to start clean from a new PR, tell me if I am now on the right track.

### CODE REVIEW

File: project_task_material_stock/models/account_analytic_line.py [hunk 1]
```diff
@@ -1,23 +1,20 @@
 # Copyright 2019 Valentin Vinagre <valentin.vinagre@qubiq.es>
+# Copyright 2020 jeo Software Jorge Obiols <jorge.obiols@gmail.com>
```
Comment (rvalyi) [inline project_task_material_stock/models/account_analytic_line.py]: I'm not sure it's well received to insert copyrights claims when you just reformat a file like that...

File: project_task_material_stock/models/stock_move.py [hunk 2]
```diff
@@ -1,23 +1,21 @@
 # Copyright 2019 Valentin Vinagre <valentin.vinagre@qubiq.es>
+# Copyright 2020 jeo Software Jorge Obiols <jorge.obiols@gmail.com>
```
Comment (rvalyi) [inline project_task_material_stock/models/stock_move.py]: I'm not sure it's well received to insert copyrights claims when you just reformat a file like that...

File: project_task_material_stock/tests/test_task_material.py [hunk 3]
```diff
@@ -1,111 +1,119 @@
 # Copyright 2015 Tecnativa - Sergio Teruel
 # Copyright 2015 Tecnativa - Carlos Dauden
 # Copyright 2016 Tecnativa - Vicent Cubells
+# Copyright 2020 jeo Software Jorge Obiols <jorge.obiols@gmail.com>
```
Comment (rvalyi) [inline project_task_material_stock/tests/test_task_material.py]: I'm not sure it's well received to insert copyrights claims when you just reformat a file like that...

**Comment by rvalyi** [comment]
Hello @jobiols it looks much better and almost right to me. Thanks for the work. Now I just put some remarks about some of your copyrights mentions, if everybody does it the way you do it doesn't really scale and it is not usually done in the OCA for trivial code reformatting like that... But module authors (not me) may comment on this better than me.

**Comment by jobiols** [comment]
Hi @rvalyi maybe you are right, I remove all the mentions except the one from CONTRIBUTORS.rst, I guess that can stay.

**Comment by jobiols** [comment]
Hello @rvalyi about the codecov. I am supposed to increase the coverate until it stop failing? it is nearly 100% yet

**Comment by hurrinico** [comment]
Hi @jobiols any news about this PR? I need to start the 14 version of it.

**Comment by dalonsod** [comment]
Hello @jobiols as @hurrinico said, should you resume this PR? Thanks!

File: project_task_material_stock/models/stock_move.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+
+class StockMove(models.Model):
+    _inherit = "stock.move"
+
+    task_material_id = fields.One2many(
+        "project.task.material", "stock_move_id", string="Project Task Material",
+    )
+
+    def _action_done(self):
```
Comment (cubells) [inline project_task_material_stock/models/stock_move.py:13]:
```suggestion
    def _action_done(self, cancel_backorder=False):
```

Method header changed in this version.

File: project_task_material_stock/models/stock_move.py [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+        # The analytical amount is updated with the value of the
+        # stock movement, because if the product has a tracking by
+        # lot / serial number, the cost when creating the
+        # analytical line is not correct.
+        res = super(StockMove, self)._action_done()
```
Comment (cubells) [inline project_task_material_stock/models/stock_move.py:18]:
```suggestion
        res = super(StockMove, self)._action_done(cancel_backorder)
```

**Review by cubells — CHANGES_REQUESTED** [review]

**Comment by pedrobaeza** [comment]
Superseded by #904 with latest requests

**Pull request closed without merging**